mesonbuild/modules/cmake.py

```diff
@@ -154,10 +154,11 @@ def dependency(self, state: ModuleState, args: T.Tuple[str], kwargs: T.Dict[str,
 
     @noKwargs
     @typed_pos_args('cmake.subproject.include_directories', str)
-    def include_directories(self, state: ModuleState, args: T.Tuple[str], kwargs: TYPE_kwargs) -> build.IncludeDirs:
+    def include_directories(self, state: ModuleState, args: T.Tuple[str], kwargs: TYPE_kwargs) -> T.List[build.IncludeDirs]:
         info = self._args_to_info(args[0])
         inc = self.get_variable(state, [info['inc']], kwargs)
-        assert isinstance(inc, build.IncludeDirs), 'for mypy'
+        assert isinstance(inc, list), 'for mypy'
+        assert isinstance(inc[0], build.IncludeDirs), 'for mypy'
```

Comment:
... Thus happily demonstrating this function has no tests. :)

Reply:
It was not my priority for Friday afternoon :) I've added one now.